compiler/plugins/input/Torch/torch-iree/InputConversion/test/auto_input_conversion.mlir

```diff
@@ -2,7 +2,7 @@
 
 // Check that the auto input conversion pipeline uses this plugin.
 
-// CHECK-LABEL: func.func @simple_add_torch
+// CHECK-LABEL: util.func public @simple_add_torch
 // CHECK:  arith.addf
 func.func @simple_add_torch(%arg0: !torch.vtensor<[2],f32>, %arg1: !torch.vtensor<[2],f32>) -> !torch.vtensor<[2],f32> {
```

Comment:
This seems like as good a time as any to bikeshed... the `util` namespace is pretty generic, so an imported program doesn't immediately stand out as using IREE dialects. I'm wondering if another name would help for this.

Reply:
someday, maybe, but unless we prefix all of our dialects with "iree_" (no thanks) I don't think there's anything else to do.